clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -29,12 +29,14 @@ def describe_clusters():
         return
 
     cluster_values = {subkey.key: subkey.value for subkey in result.leaves}
+    cluster_values = {key: cluster_values[key] for key in sorted(cluster_values)}    
```

Comment:
This won't quite do the right thing - Python dictionaries aren't ordered (see https://docs.python.org/2/tutorial/datastructures.html#dictionaries) so they're not guaranteed to remain sorted. I think you need the `sorted(cluster_values.items())`.